File: 1026java/Hw36.java
import java.util.Scanner;

public class Hw36 {
    public static void main(String[] args) {
        Scanner Scan = new Scanner(System.in);
        int count = 0;int[] result = new int[30];
        String[] input = new String[30];
        System.out.print("Enter the number : ");
        input[count] = Scan.next();
        for(count = 0; !input[count].equals("done");){
            System.out.print("Enter the number : ");
            count++;
            input[count] = Scan.next();
        }
        int sum=0;
        for(int number = 0; number < count; number++){
            result[number]= Integer.parseInt(input[number]);
            sum = sum + result[number];
        }
        int average = sum / count;
        int maximum = result[0];
        int minimum = result[0];
        for(int number =0;number <count;number++){
            if(maximum < result[number]){
                maximum = result[number];
            }
            if(minimum > result[number]){
                minimum = result[number];
            }
        }
        double[] devision = new double[count];
        double standard=0;
        for(int number = 0; number < count ; number ++){
            double down = ( average - result[number]);
            double up = 2.0;
            devision[number]= Math.pow(down,up);
            standard = standard + devision[number];
        }
        standard = standard /count;
        standard = Math.sqrt(standard);
        System.out.printf("The average is %d\nThe minimum is %d\nThe maximum is %d\nThe standard deviation is %.2f",average,minimum,maximum,standard);



    }
}

File: 1113/hw36.py
import math
list_number = []
count = 0
result = 0
number = input("Enter the number : ")
while number != "done" :
    number = int(number)
    list_number.append(number)
    number = input("Enter the number : ")
print("Numbers :",list_number)
print("The average : ",sum(list_number,0)/len(list_number))
print("The minimum : ",min(list_number))
print("The maximum : ",max(list_number))
while count < len(list_number) :
    result = result + pow((list_number[count]-(sum(list_number,0)/len(list_number))),2)
    count = count + 1
result = result / (len(list_number)-1)
result = math.sqrt(result)
print("The standard deviation is",round(result,2))

